rkt/image_list.go

```diff
@@ -167,7 +167,7 @@ func init() {
 	flagImagesSortFields = []string{importTimeField}
 	flagImagesSortAsc = true
 
-	cmdRkt.AddCommand(cmdImages)
+	cmdImage.AddCommand(cmdImages)
```

Comment:
Yes sorry. Used cmdImageList as they are global and will conflict with cmdList for listing pods.